app.js

```diff
@@ -0,0 +1,86 @@
+const School = require('./School.js');
+const Student = require('./Student.js');
+const express = require('express');
+const cors = require('cors');
+const bodyParser = require('body-parser');
+
+const port = 4000;
+const app = express();
+app.use(cors());
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: false}));
+
+
+// const classRouter = require("./Class.js");
+// const schoolRouter = require("./School.js");
+// const studentRouter = require("./Student.js");
+
+// app.use("/class", classRouter);
+// app.use("/School", schoolRouter);
+// app.use("/Student", studentRouter);
+
+let mySchool = new School();
+
+app.get("/", (req, res) => {
+    try{
+        res.json(mySchool);
+    } catch(error) {
+        console.log(error);
+    }
+});
+
+app.post("/class", (req, res) => {
+    try {
+        mySchool.addClass(req.body.name, req.body.teacher);
+        res.json({ 
+            "class": { "name": req.body.name, "teacher": req.body.teacher, "students": []},
```

Comment:
This is not technically correct. Here you are sending back your own request inside of your response. Your response should only come from your backend. In you School.js file you can `return newClass` in your `addClass` function. Then on line 34 in this function set your `mySchool.addClass(req.body.name, req.body.teacher)` to a variable, let's say newClass to keep it consistent. So `let newClass = mySchool.addClass(req.body.name, req.body.teacher)`. And then in your res.json you can set your key of `class` to the value of our new variable `newClass`. That way you are sending back a true response.